coverage/.resultset.json

```diff
@@ -0,0 +1,103 @@
+{
+  "RSpec": {
+    "coverage": {
+      "/Users/jeremiahmichlitsch/Turing/Mod_4/Projects/weather_service/spec/facades/weather_facade_spec.rb": {
+        "lines": [
+          1,
+          1,
+          1,
+          null,
+          1,
+          1,
+          1,
+          1,
+          null,
+          1,
+          1,
+          null,
+          null,
+          null
+        ]
+      },
+      "/Users/jeremiahmichlitsch/Turing/Mod_4/Projects/weather_service/app/services/map_quest_service.rb": {
+        "lines": [
+          1,
+          1,
+          4,
+          4,
+          null,
+          4,
+          null,
+          null,
+          1,
+          null,
+          1,
+          4,
+          1,
+          null,
+          null,
+          null
+        ]
+      },
+      "/Users/jeremiahmichlitsch/Turing/Mod_4/Projects/weather_service/app/facades/weather_facade.rb": {
+        "lines": [
+          1,
+          1,
+          1,
+          1,
+          null,
+          null,
+          null,
+          null,
+          1,
+          1,
+          1,
+          1,
+          null,
+          null
+        ]
+      },
+      "/Users/jeremiahmichlitsch/Turing/Mod_4/Projects/weather_service/spec/services/mapquest_service_spec.rb": {
+        "lines": [
+          1,
+          1,
+          1,
+          1,
+          1,
+          1,
+          1,
+          1,
+          1,
+          1,
+          1,
+          1,
+          1,
+          1,
+          1,
+          1,
+          1,
+          1,
+          1,
+          1,
+          1,
+          null,
+          null,
+          null,
+          1,
+          1,
+          1,
+          1,
+          1,
+          1,
+          1,
+          1,
+          null,
+          null,
+          null,
+          null
+        ]
+      }
+    },
+    "timestamp": 1621104265
+  }
+}
```

Comment:
Hey Jeremiah, it looks like the simplecov coverage files were all pushed up instead of added to the .gitignore file. Before merging this into main, let's add coverage to our .gitignore.